RELEASE-NOTES.txt

```diff
@@ -1,3 +1,9 @@
+1.4.0
+=====
+* Auto-start previously running sites on Studio launch #1166
+* Added a "What's New" popup that shows a list of new features in Studio minor and major releases
```

Comment:
```suggestion
* Added a "What's New" popup that shows a list of new features in Studio
```

Reply:
We can shorten it to avoid explaining when those can or will be shown.

Reply:
I like that proposal as well.